README.md

```diff
@@ -8,46 +8,49 @@
 
-Our design goals are:
+## Design Goals
 
+We use the following design goals as a guide when developing Slint
+
+ - **Scalable**: Run on any screen connected to a device, from desktop computers to low end embedded systems.
  - **Lightweight**: Fit into a few hundred kilobytes of RAM and require little processing power.
- - **Straightforward**: Programmers and designers should feel productive and be able to enjoy the design and development process.
-   The APIs should be consistent, easy to use, and intuitive, no matter the target language. High-quality documentation
-   should describe the APIs, teach concepts and how to use them.
- - **Native**: We support many different target platforms, from embedded devices to desktops including mobile and web. Both the user and the developer should feel at
-   home on each platform. The look and feel and experience should match the users' expectations of a
-   native application.
+ - **Intuitive**: Both developers and designers should feel productive and enjoy the design and development process.
+   The APIs should be consistent and easy to use, no matter the target language.
+ - **Native**: Slint apps should match the users' expectations of a native application. Various target platforms such as 
+   embedded devices, desktops, mobile and web should be supported so that both the user and the developer feel comfortable on
+   their platform of choice.
 
-## Quick Start
+## Get Started
 
 Check the [online demos](#demos-running-in-webassembly-simulation) or
-[build on your own system](docs/building.md).
+[build on your own system](docs/building.md). Also check out the [documentation](#documentation).
 
 ## Current Status
 
-Slint is in active development. Its state of development depends on the kind of application.
+Slint is in active development. The state of the toolkit for each platform is as follows:
 
- - **Embedded**: *Ready!* Slint is already happily used by customers on embedded devices
+ - **Embedded**: *Ready!* Slint is already being used in production projects by customers 
    (running on an Arm processor with Linux).
- - **Microcontrollers**: *Ready*. You can make GUI working on MCU with less than 300K of RAM.
+ - **Microcontrollers** (MCU): *Ready*. Slint apps can run on MCUs with less than 300K of RAM.
    [Video of Slint on STM32](https://www.youtube.com/watch?v=NNNOJJsOAis) –
    [Video of Slint on RP2040](https://www.youtube.com/watch?v=dkBwNocItGs) – 
    [docs](https://slint-ui.com/docs/rust/slint/docs/mcu/index.html)
- - **Desktop**: *In Progress*. Slint is usable on Windows, Linux and Mac. We plan on building
-   up the desktop support in subsequent releases to reach the level of integration we're aiming for.
+ - **Desktop**: *In Progress*. While Slint is usable on Windows, Linux and Mac, we are working on
+    improving the platform support in subsequent releases to reach the level of integration that
+    have aimed for.
```

Comment:
"... that *we* have aimed for." ?

Or maybe just

> While Slint is usable on Windows, Linux and Mac, we are working on improving the platform support in subsequent releases.

Reply:
Updated